libs/styles/styles.css

```diff
@@ -126,6 +126,9 @@
   --icon-size-s: 32px;
   --icon-size-xs: 24px;
   --icon-size-xxs: 16px;
+
+  /* z-index */
+  --above-all: 9000; /* Used for page tools that overlay page content */
```

Comment:
I'm not sold on this idea. What if there are multiple elements competing? I usually like to be between the `1-10` range, except when 3rd parties declare very high values that can't be overwritten otherwise.

Reply:
N/A: This was from another contributor. I've rebased and not introducing style.css changes now